spidermon/utils/zyte.py

```diff
@@ -0,0 +1,81 @@
+"""
+Module to hold a singleton to Zyte's Scrapy Cloud client.
```

Comment:
**praise** This module was improved a lot with your changes.

**suggestion** However, these changes are good, but are not related entirely with the main purpose of your PR (the `JobsComparisonMonitor`). Is it possible to rebase these changes and create a separated PR just with the changes in the hubstorage module?

Reply:
Sure, no problem. https://github.com/scrapinghub/spidermon/pull/344